pkg/codegen/jenny_go_resources.go

```diff
@@ -48,7 +48,7 @@ func (ag *ResourceGoTypesJenny) Generate(kind kindsys.Kind) (*codejen.File, erro
 	if err := tmpls.Lookup("core_resource.tmpl").Execute(buf, tvars_resource{
 		PackageName:      mname,
 		KindName:         kind.Props().Common().Name,
-		Version:          sch.Version().String(),
+		Version:          "0", // HARDCODED for now (dot not supported in agregated mode) sch.Version().String(),
```

Comment:
@toddtreece -- our "v0.0-alpha" versions worked fine in standalone, but broke when aggregated.  This hardcodes it to something without a "." -- but we should find a better pattern :)